packages/sampler-aws-xray/README.md

```diff
@@ -65,3 +68,8 @@ This package does not currently generate any attributes from semantic convention
 ## License
 
 Apache 2.0 - See [LICENSE](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/LICENSE) for more information.
+
+[license-url]: https://github.com/open-telemetry/opentelemetry-js-contrib/blob/main/LICENSE
```

Comment:
is not being used, you can remove or update the line above to actually use this one